app/routes/titiler/umd_glad_dist_alerts.py

```diff
@@ -0,0 +1,77 @@
+import os
+from datetime import date
+from typing import Optional, Tuple
+
+from dateutil.relativedelta import relativedelta
+from fastapi import APIRouter, Depends, Query, Response
+from titiler.core.resources.enums import ImageType
+from titiler.core.utils import render_image
+
+from ...models.enumerators.titiler import AlertConfidence, RenderType
+from .. import DATE_REGEX, raster_xyz
+from .algorithms.dist_alerts import DISTAlerts
+from .readers import AlertsReader
+
+DATA_LAKE_BUCKET = os.environ.get("DATA_LAKE_BUCKET")
+
+router = APIRouter()
+
+# TODO: update to the actual dataset when ready
+dataset = "dan_test"
+
+today = date.today()
+
+
+@router.get(
+    f"/{dataset}/{{version}}/dynamic/{{z}}/{{x}}/{{y}}.png",
+    response_class=Response,
+    tags=["Raster Tiles"],
+    response_description="PNG Raster Tile",
+)
+async def glad_dist_alerts_raster_tile(
```

Comment:
I wonder if we should get rid of glad_ here and in the name of the file?  So file name is just umd_dist_alerts.py and name here is dist_alerts_raster_tile.  Would be more parallel with integrated_alerts.  Also, including glad may confuse with the glad-L and glad-S2 alerts.

Reply:
i added gfw_ prefix to integrated alerts.  `{org}_optional{group}` prefix is used consistently in other route files.  GLAD  is just the name of the UMD  research group.